include/ocpp/v201/transaction.hpp

```diff
@@ -19,14 +20,23 @@ struct EnhancedTransaction : public Transaction {
     int32_t seq_no = 0;
     std::optional<float> active_energy_import_start_value;
     bool check_max_active_import_energy;
+    std::shared_ptr<DatabaseHandler> enhanced_transaction_database_handler;
```

Comment:
```suggestion
    std::shared_ptr<DatabaseHandler> database_handler;
```

Just this should be fine. It's already part of the enhanced transaction